tox.ini

```diff
@@ -18,9 +18,7 @@ changedir = docs
 commands = sphinx-build -W -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
 
 [testenv:lint]
-deps =
```

Comment:
OK, I don't feel strongly either - let's keep this version for now and see if we feel adding the dependencies in again is worth it. The Linux Travis builds usually happen in parallel, which means that as long as our lint job is < our longest job, there should be no practical impact for CI. Looking at this latest build, it seems lint goes from about 1.5 minutes to 5.5 minutes with this patch. For local testing the tox environment is cached so there's no impact (apart from first build).